js added, created footer and skills

Solodanna opened the pull request:
- Using DOM manipulation, added Footer Elemen,
- Inserted Copyright Text in Footer,
- Using DOM manipulation, created List of Skills

Solodanna committed:
js added, created footer and skills

juddin1995 commented:
You have two script tags at the bottom of your index.html. You should only have one script tag which references your js/index.js file. 

Please remove:
`<script>
  const burger = document.querySelector(".burger");
  const navLinks = document.querySelector(".nav-links");

  burger.addEventListener("click", () => {
    navLinks.classList.toggle("active");
  });
</script>`

This javascript code should be located in the js/index.js file instead.

Solodanna committed:
Burger menu code relocated to indexdex.js file

juddin1995 reviewed (approved):
Great job on completing the assignment! Here are some things you did well:

1. JavaScript File & Script Tag: The JavaScript file was correctly created as index.js and linked properly in index.html using a <script> element before the closing </body> tag.
2. DOM Manipulation: Successfully used DOM manipulation to add a footer element, create a paragraph element for copyright, and populate the skills list dynamically.
3. Date Object Usage: Correctly created the today and thisYear variables using a Date object and its method to obtain the current year.
4. CSS Flexbox & Grid: Flexbox was effectively used in the CSS file to organize the layout of the skills section and other parts of the page.
5. Responsive Design: CSS media queries are in place and demonstrate an understanding of responsive design considerations.

Areas of improvement:

1. There should only be 1 script tag which references your index.js file. You can remove the second script tag and relocate the code for the burger menu to your index.js file.
2. The nav links don't seem to be working. Please test this out and check for yourself using devtools. You can test mobile and tablet as well using the dropdown located at the top in devtools.

Solodanna committed:
- Fixed the "double script tag" issue in index.html
    - removing the duplicate closing tag,
    - relocate the code for the burger menu in index.js file.
- Added a box-shadow to the navigation bar and header section for better visual separation.
- Added smooth scrolling behavior when clicking on navigation links.
- Fixed the links in the menu to point to the correct sections.
- Adjusted the scroll padding to account for the navigation bar height.
